react-native-libraries.json

```
@@ -9746,5 +9746,18 @@
     "images": ["https://github.com/situmtech/react-native/blob/master/docs/assets/preview.png"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/ory/kratos-selfservice-ui-react-native",
+    "examples": [
+      "https://www.ory.sh/docs/getting-started/integrate-auth/react-native",
+      "https://www.ory.sh/login-react-native-authentication-example-api/"
+    ],
+    "images": ["https://www.ory.sh/docs/assets/images/1-78684ec591551f445a1064e274bf37b8.png", "https://www.ory.sh/docs/assets/images/1-78684ec591551f445a1064e274bf37b8.png"],
```

Hello @OryTiki, thank you for adding new library to the directory! 👍 

```suggestion
    "images": ["https://www.ory.sh/docs/assets/images/1-78684ec591551f445a1064e274bf37b8.png"],
```

Looks like the same asset has been listed twice, let's remove it or add an alternate image, if this was a mistake.

Other than that, new entry should be ready to ship. 🙂

---

> Looks like the same asset has been listed twice, let's remove it or add an alternate image, if this was a mistake.
Sorry, mistake on my side. Fine with the suggested change.
Thanks for the prompt reply :)